Schug service doesn't start because of missing database file

I'm trying to [launch it as a service](https://github.com/Clinical-Genomics/servers/pull/1003) on our stage server, but I get this error:

```
Mar 15 09:56:51 cg-vm1.scilifelab.se conmon[2015859]: [60B blob data]
Mar 15 09:56:22 cg-vm1.scilifelab.se systemd[1796]: schug@version_1.service: Failed with result 'exit-code'.
Mar 15 09:56:22 cg-vm1.scilifelab.se systemd[1796]: schug@version_1.service: Main process exited, code=exited, status=3/NOTIMPLEMENTED
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: [2023-03-15 09:56:22 +0100] [2] [INFO] Reason: Worker failed to boot.
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: [2023-03-15 09:56:22 +0100] [2] [INFO] Shutting down: Master
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: [2023-03-15 09:56:22 +0100] [2] [INFO] Reason: Worker failed to boot.
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: [2023-03-15 09:56:22 +0100] [2] [INFO] Shutting down: Master
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: [2023-03-15 09:56:22 +0100] [2] [WARNING] Worker with pid 5 was terminated due to signal 15
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: [2023-03-15 09:56:22 +0100] [2] [WARNING] Worker with pid 5 was terminated due to signal 15
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: [2023-03-15 09:56:22 +0100] [5] [INFO] Worker exiting (pid: 5)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: (Background on this error at: https://sqlalche.me/e/14/e3q8)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     compat.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     pool.logger.debug("Error on connect(): %s", e)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.__connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionRecord(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._create_connection()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     rec = pool._do_get()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     fairy = _ConnectionRecord.checkout(pool)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionFairy._checkout(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return fn()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3240, in _wrap_pool_connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     util.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 2097, in _handle_dbapi_exception_nocon>
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     Connection._handle_dbapi_exception_noconnection(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3243, in _wrap_pool_connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._wrap_pool_connect(self.pool.connect, _connection)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3273, in raw_connection
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     else engine.raw_connection()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 96, in __init__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._connection_cls(self, close_with_result=close_with_result)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3194, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return super(Engine, self).connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/future/engine.py", line 419, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     conn = self.connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/future/engine.py", line 393, in begin
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     with self.begin() as conn:
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/future/engine.py", line 341, in _run_ddl_visitor
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     bind._run_ddl_visitor(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/sql/schema.py", line 4746, in create_all
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     SQLModel.metadata.create_all(engine)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/app/src/schug/database/__init__.py", line 7, in <module>
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     from schug.database.session import get_session
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/app/src/schug/endpoints/exons.py", line 5, in <module>
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     from .endpoints import exons, genes, transcripts
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/app/src/schug/main.py", line 4, in <module>
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _bootstrap._gcd_import(name[level:], package, level)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     mod = importlib.import_module(module)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return util.import_app(self.app_uri)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self.load_wsgiapp()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.callable = self.load()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.wsgi = self.app.wsgi()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.load_wsgi()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     super(UvicornWorker, self).init_process()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/uvicorn/workers.py", line 66, in init_process
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     worker.init_process()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: Traceback (most recent call last):
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: The above exception was the direct cause of the following exception:
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: sqlite3.OperationalError: unable to open database file
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     compat.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     pool.logger.debug("Error on connect(): %s", e)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.__connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionRecord(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._create_connection()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     rec = pool._do_get()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     fairy = _ConnectionRecord.checkout(pool)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionFairy._checkout(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return fn()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3240, in _wrap_pool_connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: Traceback (most recent call last):
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: [2023-03-15 09:56:22 +0100] [5] [ERROR] Exception in worker process
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: [2023-03-15 09:56:22 +0100] [4] [INFO] Worker exiting (pid: 4)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: (Background on this error at: https://sqlalche.me/e/14/e3q8)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     compat.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     pool.logger.debug("Error on connect(): %s", e)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.__connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionRecord(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._create_connection()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     rec = pool._do_get()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     fairy = _ConnectionRecord.checkout(pool)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionFairy._checkout(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return fn()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3240, in _wrap_pool_connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: [2023-03-15 09:56:22 +0100] [5] [INFO] Worker exiting (pid: 5)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: (Background on this error at: https://sqlalche.me/e/14/e3q8)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return dialect.connect(*cargs, **cparams)
lines 206-238
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     compat.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     pool.logger.debug("Error on connect(): %s", e)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.__connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionRecord(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._create_connection()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     rec = pool._do_get()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     fairy = _ConnectionRecord.checkout(pool)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionFairy._checkout(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return fn()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3240, in _wrap_pool_connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: [2023-03-15 09:56:22 +0100] [5] [INFO] Worker exiting (pid: 5)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: (Background on this error at: https://sqlalche.me/e/14/e3q8)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
lines 206-239
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     compat.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     pool.logger.debug("Error on connect(): %s", e)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.__connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionRecord(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._create_connection()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     rec = pool._do_get()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     fairy = _ConnectionRecord.checkout(pool)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionFairy._checkout(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return fn()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3240, in _wrap_pool_connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: [2023-03-15 09:56:22 +0100] [5] [INFO] Worker exiting (pid: 5)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: (Background on this error at: https://sqlalche.me/e/14/e3q8)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
lines 206-243
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     compat.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     pool.logger.debug("Error on connect(): %s", e)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 666, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     self.__connect()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 371, in __init__
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionRecord(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 256, in _create_connection
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return self._create_connection()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/impl.py", line 256, in _do_get
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     rec = pool._do_get()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 476, in checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     fairy = _ConnectionRecord.checkout(pool)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 868, in _checkout
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return _ConnectionFairy._checkout(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 310, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     return fn()
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/base.py", line 3240, in _wrap_pool_connect
Mar 15 09:56:22 cg-vm1.scilifelab.se conmon[2018133]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: [2023-03-15 09:56:22 +0100] [5] [INFO] Worker exiting (pid: 5)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: (Background on this error at: https://sqlalche.me/e/14/e3q8)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]: sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) unable to open database file
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return self.dbapi.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 584, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     return dialect.connect(*cargs, **cparams)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/engine/create.py", line 590, in connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     self.dbapi_connection = connection = pool._invoke_creator(self)
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     raise exception
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:     compat.raise_(
Mar 15 09:56:22 cg-vm1.scilifelab.se podman[2018076]:   File "/home/worker/venv/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+[unreleased]
+### Fixed
+- Use a memory database as default database in demo instance
+
 [1.0.0]
 ### Added
 - Endpoint to Ensembl genes download

--- a/src/schug/config.py
+++ b/src/schug/config.py
@@ -2,6 +2,7 @@ from pathlib import Path
 
 from pydantic import BaseSettings
 
+DEMO_DB: str = "sqlite://"
 SCHUG_PACKAGE = Path(__file__).parent
 PACKAGE_ROOT: Path = SCHUG_PACKAGE.parent
 ENV_FILE: Path = PACKAGE_ROOT / ".env"
@@ -10,8 +11,7 @@ ENV_FILE: Path = PACKAGE_ROOT / ".env"
 class Settings(BaseSettings):
     """Settings for serving the schug app"""
 
-    db_name: str = "database.db"
-    db_uri: str = "sqlite:///database.db"
+    db_uri: str = DEMO_DB
     host: str = "localhost"
     port: int = 8000
 

--- a/src/schug/database/__init__.py
+++ b/src/schug/database/__init__.py
@@ -1,7 +1,11 @@
-from sqlmodel import create_engine, SQLModel
+from schug.config import DEMO_DB, settings
+from sqlmodel import SQLModel, create_engine
 
-from schug.config import settings
+DEMO_CONNECT_ARGS: dict = {"check_same_thread": False}
 
-engine = create_engine(settings.db_uri, echo=True)
+if settings.db_uri == DEMO_DB:
+    engine = create_engine(settings.db_uri, connect_args=DEMO_CONNECT_ARGS, echo=True)
+else:
+    engine = create_engine(settings.db_uri, echo=True)
 
 SQLModel.metadata.create_all(engine)

